hbase-rsgroup/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupAdminServer.java

```diff
@@ -247,23 +255,49 @@ private void moveServerRegionsFromGroup(Set<Address> servers, String targetGroup
    */
   private void moveTableRegionsToGroup(Set<TableName> tables, String targetGroupName)
```

Comment:
This method structure became very similar to _moveServerRegionsFromGroup_. Could we define a skeleton method for code reuse? Same remarks made above for  _moveServerRegionsFromGroup_ would also apply here.

Reply:
Yes, I will try.